messages.go

```diff
@@ -166,6 +166,8 @@ type Qr struct {
 	URL string `json:"u"`
 	// Session type (disclosing, signing, issuing)
 	Type Action `json:"irmaqr"`
+	// Indicator that shows whether pairing is recommended when starting the session
+	PairingRecommended bool `json:"pairingRecommended"`
```

Comment:
I expect that more people see the JSON tag name then the Go field name, don't you think it might give developers the impression that `pairing` is a required characteristic instead of a hint to the frontend? `pairingHint` might be an option and much shorter, but maybe also a bit cryptic. Is scannability and the size of the QR a known issue in some scenarios already now?

Reply:
Not necessarily a known issue, but the smaller the QR the easier it will be to scan it, right? But I agree that calling it `pairing` might be confusing. `pairingHint` is better, although indeed a little cryptic. But I don't expect that to be a problem; I think few people really look at the contents of this thing before just shoving it in a QR, and if they do, then the name might get them to lookup documentation that explains it more clearly.

Reply:
I changed it to `pairingHint` now.